jest.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    clearMocks: true,
+    collectCoverageFrom: ['src/**/*.{js,jsx,mjs}'],
+    coverageDirectory: 'coverage',
+    moduleFileExtensions: ['js', 'json', 'jsx'],
+    setupFiles: ['<rootDir>/enzyme.config.js'],
+    testEnvironment: 'jsdom',
+    testMatch: ['**/__tests__/**/*.js?(x)', '**/?(*.)+(spec|test).js?(x)'],
+    testPathIgnorePatterns: ['\\\\node_modules\\\\'],
+    testURL: 'http://localhost',
+    transformIgnorePatterns: ['<rootDir>/node_modules/'],
+    verbose: false,
+    transform: {
+        '.+\\.(css|styl|less|sass|scss)$':
+          '<rootDir>/node_modules/jest-css-modules-transform',
+        '^.+\\.js?$': 'babel-jest',
+        '^.+\\.jsx?$': 'babel-jest',
+      },
+      moduleNameMapper: {
```

Comment:
Cool. Looks like this took care of that image import error, right?

Reply:
yes, correct @Chiazokam. It returns an empty module for image extensions.